conf/flight_plans/rotorcraft_guided_flightplan.xml

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE flight_plan SYSTEM "flight_plan.dtd">
+
+<flight_plan alt="1.0" ground_alt="0" lat0="51.990634" lon0="4.376789" max_dist_from_home="20" name="Rotorcraft Optitrack (Delft)" security_height="0.3">
+  <header>
+	  #include "autopilot.h"
+    #include "guidance/guidance_h.h"
+    #include "guidance/guidance_v.h"
+    #include "modules/flight_plan_guided/flight_plan_guided.h"
```

Comment:
At least this one shouldn't be needed as modules header are automatically available for flight plans. For the other ones, if they are required by your new module, they should be included in `flight_plan_guided.h`.
As a side note, you can include modules from the flight plan itself. I means that you don't need a customized airframe that is matching a particular flight plan.